packages/x-charts-pro/src/context/ZoomProvider/Zoom.types.ts

```diff
@@ -0,0 +1,71 @@
+import { AxisId } from '@mui/x-charts/internals';
+
+export type ZoomOptions = {
+  /**
+   * The starting percentage of the zoom range. In the range of 0 to 100.
+   *
+   * @default 0
+   */
+  start?: number;
```

Comment:
Moved from `min/max` to `start/end` as echarts does, but not sure about it, both seem to work ok.

Reply:
Good for me. I don't have a strong opinion on this naming